com.unity.multiplayer.mlapi/Editor/CodeGen/NetworkBehaviourILPP.cs

```diff
@@ -170,6 +175,17 @@ public override ILPostProcessResult Process(ICompiledAssembly compiledAssembly)
 
         private bool ImportReferences(ModuleDefinition moduleDefinition)
         {
+            var debugType = typeof(Debug);
+            foreach (var methodInfo in debugType.GetMethods())
+            {
+                switch (methodInfo.Name)
+                {
+                    case k_Debug_LogWarning:
+                        if (methodInfo.GetParameters().Length == 1) Debug_LogWarning_MethodRef = moduleDefinition.ImportReference(methodInfo);
```

Comment:
I mean doesn't it just goto LogWarning anyway and just add a "[MLAPI]" prefix?

Reply:
Yes but if it is a client logging the warning (which is  the case here) then it will also send a network message to the server to display the log there as well. Since this method looks like something we might reuse later for other ILPP stuff we might want to get this right to have logging consistency? Not sure if it is really needed.